.github/workflows/staging-build-docker.yml

```diff
@@ -159,7 +159,7 @@ jobs:
       - uses: actions/checkout@v1
 
       - name: Retrieve plugin tags 
-        run:  echo "p_tag_ism=$(.github/scripts/plugin_tag.sh opendistro-for-elasticsearch/index-management)" >> $GITHUB_ENV
+        run:  echo "p_tag_ism=$(release-tools/scripts/plugin_tag.sh opendistro-for-elasticsearch/index-management)" >> $GITHUB_ENV
```

Comment:
I think the plugin_tag.sh needs to change, so that the version is picked from the manifest to be consistent.

Reply:
We will make the change later.

Reply:
Will not label this one as resolve so we can remember it.
As of now, it is working for our current setup.
Thanks.